sample/android-sdk-sample/app/src/main/java/com/bitmark/androidsdksample/fragments/AccountFragment.java

```diff
@@ -0,0 +1,128 @@
+package com.bitmark.androidsdksample.fragments;
+
+import android.app.Activity;
+import android.app.Fragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.bitmark.androidsdksample.R;
+import com.bitmark.androidsdksample.utils.Global;
+import com.bitmark.cryptography.error.ValidateException;
+import com.bitmark.sdk.features.Account;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+import static com.bitmark.androidsdksample.samples.AccountSample.createNewAccount;
+import static com.bitmark.androidsdksample.samples.AccountSample.getAccountFromRecoveryPhrase;
+import static com.bitmark.androidsdksample.samples.AccountSample.getRecoveryPhraseFromAccount;
+import static com.bitmark.androidsdksample.samples.StoreKeySample.getAccountFromKeyStore;
+import static com.bitmark.androidsdksample.samples.StoreKeySample.storeAccountToKeystore;
+import static com.bitmark.androidsdksample.utils.CommonUtil.setClipboard;
+import static com.bitmark.androidsdksample.utils.KeyUtil.getAccountNumber;
+
+public class AccountFragment extends Fragment {
+    TextView accountNumber;
+    TextView recoveryPhrase;
+    EditText recoveryPhraseInputText;
+
+    public static AccountFragment newInstance() {
+        return new AccountFragment();
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_account, container, false);
+
+        Button createNewAccountBtn = view.findViewById(R.id.createNewAccountBtn);
```

Comment:
In best practice, should bring all of commands to init the view, set listener, replace fragment or something ... in `onViewCreated()`. It happens after the view is created successfully.